client/src/partials/patients/documents/documents.js

```diff
@@ -2,28 +2,21 @@ angular.module('bhima.controllers')
 .controller('PatientDocumentsController', PatientDocumentsController);
 
 PatientDocumentsController.$inject = [
-  '$translate', '$state'
+  '$translate', '$state', 'PatientService'
 ];
 
-function PatientDocumentsController($translate, $state) {
+function PatientDocumentsController($translate, $state, Patients) {
   var vm = this;
 
-  /** breadcrumb definitions */
-  vm.paths = [{
-    label : $translate.instant('PATIENT_DOCUMENT.TITLE'),
-    current: true
-  }];
-
   /** global objects */
-  vm.patient = {};
-
-  /** expose to the view */
-  vm.setPatient  = setPatient;
+  vm.patientUuid = $state.params.patient_uuid;
 
-  /** functions definition */
-  function setPatient(patient) {
-    vm.patient = patient;
-    $state.go('patientDocuments.view', { patient_uuid: patient.uuid });
-  }
+  /** getting patient details */
+  Patients.read(vm.patientUuid)
+  .then(function (patient) {
+    vm.formatedPatientName = patient.first_name.concat(
+      ' ', patient.middle_name,
+      ' ', patient.last_name);
+  });
```

Comment:
This should have `Notify.handleError` for error handling in a `catch()` statement.